parse-zoneinfo/tests/snapshot.rs

```diff
@@ -0,0 +1,30 @@
+use std::error::Error;
+
+use insta::assert_debug_snapshot;
+
+use parse_zoneinfo::line::{Line, LineParser};
+use parse_zoneinfo::FILES;
+
+#[ignore]
```

Comment:
I added `#[ignore]` here because this test will fail every time we update the tz data. Not sure how big of a pain in the ass that will be to update? Using the `cargo-insta` tooling it is pretty easy so we might decide to just include this.